make/jdk/src/classes/build/tools/taglet/SealedGraph.java

```diff
@@ -188,12 +193,26 @@ public void addNode(TypeElement node) {
                 var styles = nodeStyleMap.computeIfAbsent(id(node), n -> new LinkedHashMap<>());
                 styles.put(LABEL, node.getSimpleName().toString());
                 styles.put(TOOLTIP, node.getQualifiedName().toString());
+                styles.put(LINK, relativeLink(rootNode, node));
                 if (!(node.getModifiers().contains(Modifier.SEALED) || node.getModifiers().contains(Modifier.FINAL))) {
                     // This indicates that the hierarchy is not closed
                     styles.put(STYLE, "dashed");
                 }
             }
 
+            // A permitted class must be in the same package or in the same module.
+            // This implies the module is always the same.
+            private static String relativeLink(TypeElement rootNode, TypeElement node) {
+                var backNavigator = rootNode.getQualifiedName().toString().chars()
+                        .filter(c -> '.' == c)
```

Comment:
"Yoda conditions" are usually frowned upon. Can `c` be `null`?

Reply:
`c` is a character